neo/Ledger/Blockchain.cs

```diff
@@ -321,7 +321,7 @@ private RelayResultReason OnNewBlock(Block block)
                     block_cache_unverified.Remove(blockToPersist.Index);
                     Persist(blockToPersist);
 
-                    if (blocksPersisted++ < blocksToPersistList.Count - 2) continue;
+                    if (blocksPersisted++ < blocksToPersistList.Count - (2 + Math.Max(0,(15 - SecondsPerBlock))) continue;
```

Comment:
The concept looks alright, but you are missing a right parenthesis. 

Maybe there should be a setting for how many seconds worth of blocks should be relayed for this case. So right now with 15 second blocks it is covering 30 seconds worth, since it is using 2, so the default setting would be 30 seconds.

With that default, 10 second blocks relay 3 blocks, 5 second blocks relay 6 blocks, etc. someone can easily change the setting if they want behavior to forward blocks after longer network outage.

Reply:
Thanks for the parenthesis.

I am glad you like.
If you think it is correct let's merge it like this because it will help us to improve the issue, in the future we add this additional features and parameters.